docs/developer-guide/rosplane/estimator/estimator-example.md

```diff
@@ -0,0 +1,373 @@
+$N$ is typically 10, meaning that the previous estimate that was updated by a measurement is updated in 10 steps instead of a single step to calculate the estimate at the current time step.
+The length of each of the $N$ steps is 1/$N$ the original time step.
+
+The attitude estimation is propagated according to the model $f$:
+
+\begin{equation}
+    f =
+    \begin{bmatrix}
+        \dot{\phi} = p + (q \sin{\phi} + r \cos{\phi}) \tan{\theta} \\
+        \dot{\theta} = q \cos{\phi} + r \sin{\phi})
+    \end{bmatrix}
+\end{equation}
+
+This propagated estimate is then used in the calculation of the Jacobian $A$.
+
+\begin{equation}
+    A =
+    \begin{bmatrix}
+        (q \cos{\phi} - r \sin{\phi})\tan{\theta} & \frac{q\sin{\phi} + r\cos{phi}}{\cos^2{\theta}} \\
+        0 & -q \sin{\phi} - r \cos{\phi} \\
+    \end{bmatrix}
+\end{equation}
+
+This Jacobian is then used to find a second-order approximation of the matrix exponential, $A_d$.
+
+\begin{equation}
+    A_d = I + \frac{T_s}{N} A + \frac{1}{2} \frac{T_s^2}{N^2} A^2
+\end{equation}
+
+Where $T_s$ is the length of a time step.
+$A_d$ is then used to propagate the actual covariance of the estimate.
+
+The process noise due to model uncertainty is defined in the matrix $Q$, but the process uncertainty due to the use of the rate gyro measurements $Q_g$, has not been adjusted for yet.
+This is done with the use of the matrix $G$ which takes into account the Coriolis effects of the gyro measurements.
+The measurement variance is transformed into process noise due to gyro measurements.
+All of these contribute into finding the current covariance of our estimate, $P$.
+This is done by the following equation:
+
+$$P_a = A_d P A_d^\top + (Q + G Q_g G^\top) \frac{T_s^2}{N^2}$$
+
+With this propagated estimate and covariance we are now ready for a measurement update.
+
+### Measurement Update
+
+A measurement update provides a check on our propagated estimate and we take this new information and fuse it into our estimate.
+The Kalman filter allows us to optimally adjust our estimate, our tuned process noises, and the noise characteristics of our sensor given the measurement.
+These noise characteristics are captured in a diagonal matrix, $R_{sensor}$.
+The entries are the variances and for the attitude step this is defined as:
+
+\begin{equation}
+    R_{accel} = 
+    \begin{bmatrix}
+        \sigma_{accel, x}^2 & 0 & 0 \\
+        0 & \sigma_{accel, y}^2 & 0 \\
+        0 & 0 & \sigma_{accel, z}^2 \\
+    \end{bmatrix}
+\end{equation}
+
+Using our estimate and a model set of equations $h$, we predict the measurements the accelerometer will produce.
+We will then compare the actual and predicted measurements and optimally adjust our estimate with the new information.
+Since measurements come in much faster than the model propagates the measurement step is run every time the propagated estimate is calculated.
```

Comment:
This isn't true if we crank up the `frequency` parameter, right?

Reply:
Ya maybe this is the spot to put in the caveat we talked about....

Reply:
I added the explanation in the newest commit, let me know your thoughts.